config/routes.rb

```diff
@@ -27,6 +27,11 @@
                  constraints: ->(_) { FeatureFlipper.show_education_benefit_form? }
     end
 
+    resource :health_care_application, only: [:create, :index] do
+      post :create
+      get :index
```

Comment:
Just figured out that's because I need to specify `resources` instead of just `resource`, but now I'm getting a Rails error (`uninitialized constant V0::HealthCareApplicationController`) when trying to go to that url.